app/src/androidTest/java/com/g11x/checklistapp/LocalRepositoryTest.java

```diff
@@ -0,0 +1,25 @@
+package com.g11x.checklistapp;
+
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+import android.test.ProviderTestCase2;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/** Tests for the app's ContentProvider.*/
+@RunWith(AndroidJUnit4.class)
+public class LocalRepositoryTest extends ProviderTestCase2<LocalRepository> {
+    public LocalRepositoryTest() {
+        super(LocalRepository.class, Constants.LOCAL_REPOSITORY_AUTHORITY);
```

Comment:
@PaulRashidi Is there a way to access the resources strings from tests? Trying to figure that out so I can use `R.strings.content_provider_authority`, but so far it isn't clear to me how to.